__tests__/cmds/docs.test.js

```diff
@@ -359,7 +392,12 @@ describe('rdme docs:edit', () => {
     const slug = 'getting-started';
     const body = 'abcdef';
 
-    nock(config.host).get(`/api/v1/docs/${slug}`).reply(200, { body });
+    nock(config.host)
```

Comment:
Does this mock need to get cleaned up with `.done()` in the `.catch()`?

Reply:
Oh interesting, yes it does! Will fix 👍.